client/ayon_blender/plugins/publish/validate_absolute_external_data_paths.py

```diff
@@ -0,0 +1,61 @@
+import bpy
+import pyblish.api
+from ayon_core.pipeline import (
+    OptionalPyblishPluginMixin
+)
+from ayon_core.pipeline.publish import (
+    RepairAction,
+    ValidateContentsOrder,
+    PublishValidationError
+)
+
+
+
+class ValidateAbsoluteExternalDataPaths(pyblish.api.InstancePlugin,
+                                        OptionalPyblishPluginMixin):
+    """Validates Absolute External Data Paths
+
+    This validator checks if all external data paths are absolute
+    to ensure the links would not be broke when publishing
+    """
+
+    label = "Validate Absolute External Data Paths"
+    order = ValidateContentsOrder
+    hosts = ["blender"]
+    optional = True
+    actions = [RepairAction]
```

Comment:
This runs over all instances, but seems to do a scene-wide check not related to a particular instance? So it should instead be a ContextPlugin to avoid running the same check multiple times.

Plus, making this optional would mean you'd only need to disable once then instead of one each instance (where it would be validating the same thing).

So, either:
- `InstancePlugin` -> `ContextPlugin` - you may need a `RepairContextAction`
- Make this instance specific and only validate data blocks relevant to the instance.